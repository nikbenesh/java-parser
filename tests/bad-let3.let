let x := 4 * 2 in
