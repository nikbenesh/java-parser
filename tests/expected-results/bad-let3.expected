Error (2): Unexpected token EOF
Parse failed.
